python/lsst/verify/extract_metricvalues.py

```diff
@@ -43,7 +43,8 @@ def print_metrics(butler, kind, *, data_id_keys=None,
         for example: ``("detector", "visit")``.
     data_id_restriction : `dict`, optional
         Only include values whose dataId matches these key:value pairs;
-        for example: ``{"detector": 50}``.
+        for example: ``{"detector": 50}``. If a metric does not have a key in
+        its DataCoordinate, it is not included.
```

Comment:
Strictly speaking, metrics don't have DataCoordinates, only metric values do. Maybe "If a metric does not use a key"?